developers/wallets/native.md

```diff
@@ -97,8 +97,7 @@ The endpoint declared there (`/switchboard` in the example above) will receive a
 Here is an example URL showing the `request` query parameter and its URL encoded value:
 
 <pre><code style="white-space: normal;overflow: auto;word-break: break-word;">
```

Comment:
`break-word` is apparently [deprecated](https://drafts.csswg.org/css-text/#word-break-property).  Perhaps dense strings like this should use `break-all`?